go/runtime/registry/notifier_rofl.go

```diff
@@ -0,0 +1,337 @@
+package registry
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/oasisprotocol/oasis-core/go/common/cbor"
+	"github.com/oasisprotocol/oasis-core/go/common/logging"
+	consensus "github.com/oasisprotocol/oasis-core/go/consensus/api"
+	roothash "github.com/oasisprotocol/oasis-core/go/roothash/api"
+	"github.com/oasisprotocol/oasis-core/go/runtime/bundle/component"
+	"github.com/oasisprotocol/oasis-core/go/runtime/host"
+	"github.com/oasisprotocol/oasis-core/go/runtime/host/composite"
+	"github.com/oasisprotocol/oasis-core/go/runtime/host/protocol"
+	"github.com/oasisprotocol/oasis-core/go/runtime/transaction"
+)
+
+const (
+	// roflNotifyTimeout is the maximum amount of time runtime notification handling can take.
+	roflNotifyTimeout = 2 * time.Second
+
+	/// methodGetConfig is the name of the `get_config` method.
+	methodGetConfig = "get_config"
```

Comment:
Should this be namespaced? Eg. `rofl.GetConfig` or similar.

Also it belongs to `go/runtime/rofl/api`.